doc/src/learn/exchange-integration-guide.md

```diff
@@ -0,0 +1,238 @@
+
+ * [sui_payAllSui](https://docs.sui.io/sui-jsonrpc#sui_payAllSui)
+   This method accepts an array of SUI token IDs. It merges all existing tokens into one, deducts the gas fee, then sends the merged token to the recipient address.
+
+   The method is especially useful if you want to transfer all SUI from an address. To merge together all coins for an address, set the recipient as the same address. This is a native Sui method so is not considered a transaction in Sui.
+
+ * [sui_paySui](https://docs.sui.io/sui-jsonrpc#sui_paySui)
+   This operation accepts an array of SUI token IDs, an array of amounts, and an array of recipient addresses.
+
+   The amounts and recipients array map one to one. Even if you use only one recipient address, you must include it for each amount in the amount array.
+
+   The operation merges all of the tokens provided into one token object and settles the gas fees. It then splits the token according to the amounts in the amounts array and sends the first token to the first recipient, the second token to the second recipient, and so on. Any remaining SUI on the token stays in the source address.
+
+   The benefits of this method include: no gas fees for merging or splitting tokens, and the abstracted token merge and split. The `sui_paySui` operation is a native function, so the merge and split operations are not considered Sui transactions. The gas fees for them match typical transactions on Sui.You can use this operation to split coins in your own address by setting the recipient as your own address. Note that the total value of the input coins must be greater than the total value of the amounts to send.
+
+ * [sui_pay](https://docs.sui.io/sui-jsonrpc#sui_pay)
+   This method is similar to sui_paySui, but it accepts any kind of coin or token instead of only SUI. You must include a gas token, and all of the coins or tokens must be the same type.
+
+ * [sui_transferSui](https://docs.sui.io/sui-jsonrpc#sui_transferSui)
+    This method accepts only one SUI token object and an amount to send to the recipient. It uses the same token for gas fees, so the amount to transfer must be strictly less than the value of the SUI token used.
+
+## SUI Staking and Delegation
+
+The Sui blockchain uses a delegated Proof-of-Stake mechanism (DPoS). This allows SUI token holders to delegate their tokens to any validator of their choice. When someone delegates their SUI tokens, it means those tokens are locked for the entire epoch. Users can withdraw their stake and stake with a different validator between epochs.
+
+SUI holders who delegate their tokens to validators earn rewards for helping secure the Sui  network. Sui determines rewards for delegation based on stakw rewards on the network, and distributes them at the end of each epoch.
+
+The total stake delegated to a validator determines the validator’s voting power.
```

Comment:
The total voting power in the Sui Network is always, 10,000, so individual voting powers can be interpreted as basis points (e.g., voting power 101 = 1.01%). Sui's quorum threshold (number of votes needed to confirm a transaction) is 6,667 (i.e., >2/3). The voting power for a single validator is capped at 1,000 (i.e., 10%) regardless of how much stake the validator has.

Reply:
Added this comment to source doc